timesketch/frontend-ng/src/components/Explore/EventActionMenu.vue

```diff
@@ -14,24 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <template>
-  <v-menu
-    v-model="showMenu"
-    offset-y
-    transition="slide-y-transition"
-  >
+  <v-menu v-model="showMenu" offset-y transition="slide-y-transition">
     <template v-slot:activator="{ on, attrs }">
-      <v-icon v-bind="attrs" v-on="on" class="ml-1">mdi-dots-vertical</v-icon>
+      <v-icon v-bind="attrs" v-on="on" class="ml-2">mdi-dots-vertical</v-icon>
```

Comment:
This change with the removal of the margin in the EventTagMenu.vue file will result in star and tag icon being very close to each other and the three dot submenu icon standing separate and "somewhat alone". Visually I personally prefer the even spacing between all three icons.

Reply:
Comparison:  
![icons_a](https://user-images.githubusercontent.com/99879757/219021016-2b358e95-b727-41b4-ac78-762c4f12114b.png)        ![icons_b](https://user-images.githubusercontent.com/99879757/219021043-a636edd5-d63a-4dad-b1e1-2d64323eecb9.png)

Reply:
Food catch! This was unintentional. I had to fix the tag menu margin as well to line them up. This is fixed now.